clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/CreateGoogleHttpLoadBalancerAtomicOperation.groovy

```diff
@@ -57,15 +54,18 @@ class CreateGoogleHttpLoadBalancerAtomicOperation  implements AtomicOperation<De
   }
 
   /**
-   * curl -X POST -H "Content-Type: application/json" -d '[{ "createGoogleHttpLoadBalancerDescription": {"credentials": "my-account-name", "loadBalancerName": "http-lb", "googleHttpLoadBalancer": {"name": "http-lb", "defaultService": {"name": "default-backend-service", "backends": [], "healthCheck": {"name": "basic-check", "requestPath": "/", "port": 80}}, "certificate": "cert-name", "hostRules": [{"hostPatterns": ["host1.com", "host2.com"], "pathMatcher": {"pathRules": [{"paths": ["/path", "/path2/more"], "backendService": {"name": "backend-service", "backends": [], "healthCheck": {"name": "basic-check", "requestPath": "/", "port": 80}}}] "defaultService": {"name": "pm-backend-service", "backends": [], "healthCheck": {"name": "basic-check", "requestPath": "/", "port": 80}}}}]}}}]' localhost:7002/ops
+   * curl -v -X POST -H "Content-Type: application/json" -d '[{ "createGoogleHttpLoadBalancerDescription": {"credentials": "my-google-account", "googleHttpLoadBalancer": {"name": "http-create", "portRange": "80", "defaultService": {"name": "default-backend-service", "backends": [], "healthCheck": {"name": "basic-check", "requestPath": "/", "port": 80, "checkIntervalSec": 1, "timeoutSec": 1, "healthyThreshold": 1, "unhealthyThreshold": 1}}, "certificate": "", "hostRules": [{"hostPatterns": ["host1.com", "host2.com"], "pathMatcher": {"pathRules": [{"paths": ["/path", "/path2/more"], "backendService": {"name": "backend-service", "backends": [], "healthCheck": {"name": "health-check", "requestPath": "/", "port": 80, "checkIntervalSec": 1, "timeoutSec": 1, "healthyThreshold": 1, "unhealthyThreshold": 1}}}], "defaultService": {"name": "pm-backend-service", "backends": [], "healthCheck": {"name": "derp-check", "requestPath": "/", "port": 80, "checkIntervalSec": 1, "timeoutSec": 1, "healthyThreshold": 1, "unhealthyThreshold": 1}}}}]}}}]' localhost:7002/ops
```

Comment:
Can you please also add a sample command with the minimal set of required attributes?